js/exporter/exceljs/export.js

```diff
@@ -247,15 +268,26 @@ export const Export = {
                 this.setAlignment(excelCell, wrapText, horizontalAlignment);
             }
 
-            if(isFunction(customizeCell)) {
-                customizeCell(privateOptions._getCustomizeCellOptions(excelCell, cell));
-            }
-
             if(privateOptions._needMergeRange(rowIndex, headerRowCount)) {
-                const mergeRange = this.tryGetMergeRange(rowIndex, cellIndex, mergedCells, dataProvider);
-                if(isDefined(mergeRange)) {
-                    mergeRanges.push(mergeRange);
+                if(this.cellInMergeRanges(mergedCells, rowIndex, cellIndex)) {
+                    const cell = mergedCells[rowIndex][cellIndex];
```

Comment:
imho we can combine 'cellInMergeRanges' with 'mergedCells[rowIndex][cellIndex]' into 'mergedCell = _tryGetMergedCell()'

Reply:
9c50f59

Reply:
This will cause additional confusion of logic in one method. Inside, along with updating the range, there will be an operation to copy the value and styles from the master cell.